frontend/src/router/guards.js

```diff
@@ -55,6 +55,10 @@ export function createGlobalBeforeGuards () {
       if (expiresIn > 0) {
         clearTimeout(sessionTimeoutId)
         sessionTimeoutId = setTimeout(() => {
+          if (!authnStore.isExpired()) {
+            logger.info('User session updated by other browser process --> Skipping signout')
+            return
+          }
```

Comment:
shouldn't the session timeout be restarted?

Reply:
Removed the sessionTimeout timer